public/app/features/explore/Explore.tsx

```diff
@@ -21,7 +21,11 @@ import {
   ExploreUIState,
   ExploreUrlState,
   LogsModel,
+  NavModel,
 } from '@grafana/data';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { Branding } from 'app/core/components/Branding/Branding';
+import { getTitleFromNavModel } from 'app/core/selectors/navModel';
```

Comment:
nit: I think it would make sense to import getNavModel from here as well. Or the other way around.